source/documentation/free_radius_presentation.md

```diff
@@ -0,0 +1,94 @@
+# FreeRADIUS Notes
+
+## AAA
+
+GovWifi currently has a set of 6 RADIUS servers. These servers control network access with [AAA authentication][aaa link].
+![alt text][radius_exchange]
+
+### Authentication
+
+- Runs on port 1812.
+- Requests user details stored in the database via the [Authentication API][authentication api link].
+
+### Authorisation
+
+- We do not have different levels of access so this is a NOOP.
+
+### Accounting
+
+- Runs on Port 1813.
+- Disabled for GovWifi, network admins can monitor this on their own infrastructure.
```

Comment:
It's not actually disabled, we just don't do anything with the data.

Might be worth clarifying here